docs/reference/remap.cue

```diff
@@ -360,6 +360,36 @@ remap: {
 			]
 		}
 
+		"Blocks": {
+			href: "blocks"
+
+			description: """
+				TRL supports organizing expressions into blocks using curly braces. Everything
+				inside of a block is evaluated as a single expression. In this example, the value
+				assigned to the variable `$success` is `true` if the value of the `status_code`
+				field is `201`:
```

Comment:
I've considered removing support for blocks, outside control-flow statements such as if-statements.

Because we don't support variable scoping, there isn't any difference between these two:

```rust
$success = {
	$success_codes = [200, 201, 202, 204]
	$code = .status_code
	contains($success_codes, $code)
}
```

```rust
$success_codes = [200, 201, 202, 204]
$code = .status_code
$success = contains($success_codes, $code)
```

Other than maybe syntactic preference (at the expense of potentially confusing people who expect variable scoping to exist).

Reply:
I'll leave this section for now and we can remove it on a follow-up PR if the story changes